src/components/Reports/GDPR/SubscriberGdpr.js

```diff
@@ -0,0 +1,201 @@
+                      <div className="dp-calification">
+                        <span className="dp-useremail-campaign">
+                          <strong>{state.subscriber.email}</strong>
+                        </span>
+                        <StarsScore score={state.subscriber.score} />
+                      </div>
+                      <span className="dp-username-campaing">
+                        {state.subscriber.firstName ? state.subscriber.firstName.value : ''}{' '}
+                        {state.subscriber.lastName ? state.subscriber.lastName.value : ''}
+                      </span>
+                      <span className="dp-subscriber-icon">
+                        <span
+                          className={
+                            'ms-icon icon-user ' +
+                            getSubscriberStatusCssClassName(state.subscriber.status)
+                          }
+                        ></span>
+                        <FormattedMessage id={'subscriber.status.' + state.subscriber.status} />
+                      </span>
+                      {state.subscriber.status.includes('unsubscribed') ? (
+                        <ul className="dp-rowflex col-sm-12 dp-subscriber-info">
+                          <li className="col-sm-12 col-md-4 col-lg-3">
+                            <span className="dp-block-info">
+                              <FormattedMessage id="subscriber_history.unsubscribed_date" />
+                            </span>
+                            <span>
+                              <FormattedDate value={state.subscriber.unsubscribedDate} />
+                            </span>
+                          </li>
+                        </ul>
+                      ) : null}
+                    </div>
+                  </header>
+                  <div>
+                    <div className="dp-table-responsive">
+                      <table
+                        className="dp-c-table"
+                        aria-label={_('subscriber_history.table_result.aria_label_table')}
+                        summary={_('subscriber_history.table_result.aria_label_table')}
+                      >
+                        <thead>
+                          <tr>
+                            <th scope="col">
+                              <FormattedMessage id="subscriber_gdpr.permission_name" />
+                            </th>
+                            <th scope="col">
+                              <FormattedMessage id="subscriber_gdpr.permission_description" />
+                            </th>
+                            <th scope="col">
+                              <FormattedMessage id="subscriber_gdpr.permission_value" />
+                            </th>
+                          </tr>
+                        </thead>
+                        <tbody>
+                          {state.subscriber.fields.some(
+                            (element) =>
```

Comment:
need this validation if you filter after?

Reply:
I want to show a message in case we have no permissions in between the fields.

Reply:
Fixed!